fendermint/testing/contracts/QueryBlockhash.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+contract QueryBlockhash {
+    constructor() {}
+
+    function getBlockhash(uint blockNumber) public view returns (bytes32) {
```

Comment:
How about we make the argument an offset, call NUMBER, add the offset, then call BLOCKHASH? This would make the test less time sensitive. And you'd pass in a negative block number to look back (which means this argument type would have to be int)

Reply:
I prefer absolute number here, it make comparing this result to the API simpler